debugAdapterProtocol.json

```diff
@@ -3352,6 +3360,10 @@
 				"supportsANSIStyling": {
 					"type": "boolean",
 					"description": "The debug adapter supports ANSI escape sequences in styling of `OutputEvent.output` and `Variable.value` fields."
+				},
+				"supportsGotoInstruction": {
```

Comment:
I would name this `supportsGotoInstructionTargets` just in case we ever have another type of 'go to' action